molo/forms/tests/test_reaction_questions.py

```diff
@@ -0,0 +1,528 @@
+        self.client.login(username='super', password='password')
+
+        response = self.client.get('/admin/')
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, 'reactionquestion')
+
+    def test_user_has_perm_can_see_reaction_question_modeladmin(self):
+        self.client.login(username='username', password='password')
+        user = User.objects.filter(username='username').first()
+
+        # User shoudn't see the reaction question model admin
+        self.assertTrue(user.has_perm('wagtailadmin.access_admin'))
+        self.assertFalse(user.has_perm('forms.can_view_response'))
+
+        response = self.client.get('/admin/')
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, 'reactionquestion')
+
+        # User shoud see the reaction question model admin
+        self.test_group.permissions.add(self.reaction_question)
+        user = User.objects.filter(username='username').first()
+        self.assertIn(self.test_group, user.groups.all())
+
+        self.assertTrue(user.has_perm('wagtailadmin.access_admin'))
+        self.assertTrue(user.has_perm('forms.can_view_response'))
+
+        response = self.client.get('/admin/')
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, 'reactionquestion')
+
+
+class TestReactionQuestionChoiceFeedbackView(FormsTestCase, TestCase):
+    """Test response reaction """
+
+    def setUp(self):
+        self.mk_main()
+        self.english = SiteLanguageRelation.objects.create(
+            language_setting=Languages.for_site(self.main.get_site()),
+            locale='en',
+            is_active=True)
+
+        self.assertTrue(self.section_index)
+        self.yourmind = self.mk_section(
+            self.section_index, title='Your mind')
+        self.reaction_index = self.get_reaction_index(self.main)
+
+    def test_view(self):
+        article = self.mk_article(self.yourmind)
+        question = self.mk_reaction_question(self.reaction_index, article)
+        choice = ReactionQuestionChoice(title='yes', success_message='Yes!!!')
+        question.add_child(instance=choice)
+        choice.save_revision().publish()
+
+        choice2 = ReactionQuestionChoice(title='no', success_message='No!!!')
+        question.add_child(instance=choice2)
+        choice2.save_revision().publish()
+
+        kw = {
+            'article_slug': article.slug,
+            'question_id': question.pk,
+            'choice_slug': choice.slug,
+        }
+        url = reverse('molo.forms:reaction-feedback', kwargs=kw)
+
+        res = self.client.get(url)
+        self.assertTrue(res.status_code, 200)
+        self.assertEqual(
+            res.template_name,
+            ['forms/reaction_question_feedback.html'])
+        self.assertContains(res, article.title)
+        self.assertContains(res, choice.success_message)
+
+    def test_load_user_choice_reaction_question(self):
```

Comment:
This test and the one below it aren't related to the ReactionQuestionChoiceFeedbackView are they?
Maybe they should be in their own class?